<launch>
  <arg name="kinect_name" default="camera"/>

  <node name="kinect_humand_detection" pkg="kinects_human_tracking" type="kinect_human_detection" output="screen">
    <param name="kinect_topic_name" value="$(arg kinect_name)/background_sub" />
    <param name="robot_topic_name" value="robot_model_to_pointcloud/robot_cloud2" />
    <param name="clusters_topic_name" value="kinect_both/clusters" />
    <param name="out_topic_name" value="kinect_both/human_detection" />
  </node>
</launch>
